assets/styles.css

```diff
@@ -0,0 +1,73 @@
+.header-wrapper {
+  height: 100px;
+  padding: 15px 10px;
+  color: white;
+  background-color: #f72210;
+}
+
+.logo-head {
+  color: #fff;
+  fill: white;
+  width: 400px;
+}
+
+.header-font {
+  font: bold 13px/24px "Roboto Condensed", sans-serif;
+  font-size: 1rem;
+  font-weight: 400;
+  line-height: 1.5;
+}
+
+.rounded-pill {
+  background-color: #ffe400;
+  height: 30px;
+}
+
+@media screen and (max-width: 992px) {
+  .header-wrapper {
+    height: 50px;
+    color: white;
+    background-color: #f72210;
+    max-width: 1310px;
+  }
+
+  .logo-head {
+    color: #fff;
+    fill: white;
+    width: 150px;
+  }
+}
+
+/* FOOTER CSS */
+
+.footer-links {
+  font-size: 14px;
+  line-height: 15px;
+}
+
+.footer-section {
+  border-top: 1px solid #666;
+}
+
+.logo-footer {
+  color: #fff;
```

Comment:
- [ ] Use bootstrap classes to minize the custom stylesheet. This are not all. But give a review to the CSS stylesheet and try to use as many Bootstrap classes as possible.
  - [ ] [floats](https://getbootstrap.com/docs/4.4/utilities/float/)
  - [ ] [text-color](https://getbootstrap.com/docs/4.4/utilities/colors/#color)
  - [ ] [text transform](https://getbootstrap.com/docs/4.4/utilities/text/#text-transform)